package.json

```diff
@@ -4,15 +4,13 @@
     "description": "RESTful API for performance engineering",
     "scripts": {
         "start": "node ./server",
-        "test": "mocha test/**/*.js",
+        "test:mocha": "mocha test/**/*.js",
         "eslint": "eslint -c .eslintrc src routes public/js",
-        "publish": "npm run eslint && npm run test",
-        "semantic-release": "semantic-release",
-        "travis-deploy-once": "travis-deploy-once"
+        "test": "npm run eslint && npm run test:mocha"
```

Comment:
The recommendation I was given over on `orglinter` was to use:

```json
    "lint": "eslint .",
    "test:mocha": "mocha",
    "test": "nyc npm run test:mocha",
    "posttest": "npm run lint"
```

Do you think that would work here, since you're changing around your test command setup?